recipes/datarobot/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "datarobot" %}
+{% set version = "2.8.1" %}
+{% set sha256 = "338f5233a48251ea6aafae858f22e25e1610c55a20b17aafd92ef9da24cd5607" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - pip
+    - pytest-runner
+  run:
+    - python
+    - pandas >=0.15
+    - pyyaml >=3.11
+    - requests-toolbelt >=0.6
+    - trafaret <1.1,>=0.7
+
+test:
+  imports:
+    - datarobot
+
+about:
+  home: https://www.datarobot.com/
+  license: Apache-2.0
+  license_family: Apache
+  license_file: LICENSE.txt
+  summary: 'This client library is designed to support the DataRobot API.'
+
+  description: |
+    datarobot is a client library for working with the <https://www.datarobot.com/> platform API.
+  doc_url: https://datarobot-public-api-client.readthedocs-hosted.com
+
+extra:
+  recipe-maintainers:
+    - elatt
+    - dsakagi
+    - peterhurford
```

Comment:
@peterhurford do you agree to be a maintainer of this package?

Reply:
Yes